### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by Koshao-sys**
Hi, 

**I worked on the following in this Pull request;**

- Add RSpec as a gem to your project.
- Create a Request spec file for all your controllers.
- Make sure that for each action you check: 
  -  If response status was correct.
  - If a correct template was rendered.
  - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/users_spec.rb
Comment (fobadara) [inline spec/requests/users_spec.rb:1]: [REQUIRED] Kindly ensure that the correct view template is rendered with `render_template`

File: spec/requests/posts_spec.rb
Comment (fobadara) [inline spec/requests/posts_spec.rb:1]: [REQUIRED] Kindly ensure that the correct view template is rendered with `render_template`

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Koshao-sys, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Good use of `describe` and `it` blocks

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @Koshao-sys     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**